zulipterminal/ui_tools/boxes.py

```diff
@@ -1200,16 +1200,16 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
             #   ```quote
             #   message_content
             #   ```
-            quote = '@_**{}|{}** '.format(self.message['sender_full_name'],
-                                          self.message['sender_id'])
-
-            # FIXME: The type and generation of link needs to be discussed.
-            # For interoperability webapp link should be given, but
-            # that would not narrow to the quoted message inside ZT.
             absolute_url = absolute_url_of_message(self.model, self.message)
-            quote += '[said]({}):\n'.format(absolute_url)
-            quote += '```quote\n' + self.model.client.get_raw_message(
-                self.message['id'])['raw_content'] + '\n```\n'
+            fence = get_unused_fence(self.model.client.get_raw_message(
+                self.message['id'])['raw_content'])
```

Comment:
If you want to put this addition as the first commit we can merge this separately.

However, it would be useful to have some tests of the introduced function (and perhaps the integrated case) in the same commit too.